recipe/meta.yaml

```diff
@@ -9,15 +9,10 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 030644df4611007ff7dc962d981f390361e6c97a34e5cbc393ddfbe019ffe2c1
-  patches:
-    # https://github.com/lz4/lz4/pull/962
-    - patches/0002-include-local-header-directory.patch  # [win]
-    # See: https://github.com/lz4/lz4/pull/972
-    - patches/CVE-2021-3520.patch
+  sha256: 0b0e3aa07c8c063ddf40b082bdf7e37a1562bda40a0ff5272957f3e987e0e54b
 
 build:
-  number: 1
+  number: 0
 
 requirements:
   build:
```

Comment:
`patch  # [unix]` and `m2-patch  # [win]` can be removed as we do not apply any patches